documentation/how-to-topics/Wcf-service-with-VBnet.md

```diff
@@ -0,0 +1,173 @@
+# Using WCF with VB.Net
+
+## Introduction
+
+VB.Net is only supported in OpenSilver 2.0+ (including 2.0 preview). It is not supported by the earlier versions.
```

Comment:
Maybe you should mention that the .Browser and .Simulator projects are in C#, but they are only bootstrap helper projects that 99% of time don't need to be changed by the VB developer.